contracts/PvpArena.sol

```diff
@@ -57,6 +57,8 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
     uint8 public wageringFactor;
     /// @dev percentage of duel cost charged when rerolling opponent
     uint256 public reRollFeePercent;
+    /// @dev percentage of entry wager charged when withdrawing from arena with pending duel
+    uint256 public withdrawFeePercent;
```

Comment:
you cannot change the order of state variables. New ones have to be added after all the others